ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/validation/CommandsValidator.scala

```diff
@@ -223,8 +224,13 @@ final class CommandsValidator(
     } yield Submitters(actAs, readAs)
   }
 
+  // TODO: Address usage of deprecated class DeduplicationTime
+
   /** We validate only using current time because we set the currentTime as submitTime so no need to check both
     */
+  @nowarn(
+    "msg=class DeduplicationTime in object DeduplicationPeriod is deprecated"
```

Comment:
Please use `cat=deprecation&origin=^com\.daml\.ledger\.api\.v1\.commands\.Commands\.DeduplicationPeriod\.DeduplicationTime$` in the future.

Reply:
Now I finally understand how that one works, thanks!